static/sass/patterns/_text-blocks.scss

```diff
@@ -0,0 +1,53 @@
+// Text Blocks
+// ===========
+
+.text-block {
+  @include contentblock;
+}
+
+.textblock-headline {
+  @include contentblock-highlight('normal');
+}
+
+.textblock-content {
+  @include contentblock-copy;
+
+  p {
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+}
+
+.headline-contrast {
+  margin-top: 0;
+
+  @include above('small-page') {
+    text-align: right;
+  }
+
+  span {
+    display: block;
+  }
+
+  .title-main {
```

Comment:
Not sure how much we gain by reusing this class, vs what we lose by using it in a new / altered way?

Reply:
true, I can adjust.